Dockerfile.rhel8

```diff
@@ -23,7 +23,7 @@ COPY requirements/requirements.insights.txt /tmp/requirements.txt
 RUN set -ex; \
     DNF="dnf -y --disableplugin=subscription-manager" && \
     INSTALL_PKGS="glibc-langpack-en git-core libpq python38" && \
-    INSTALL_PKGS_BUILD="gcc libpq-devel python38-devel" && \
+    INSTALL_PKGS_BUILD="gcc libpq-devel python38-devel openldap-devel" && \
```

Comment:
Not sure if it's  good idea to add this as a dependency to our production image, since I believe this just gets used on console.redhat.com.

@awcrosby ^

Reply:
Is this required since we're adding django ldap as a python dep?

Reply:
Yes it is required because we import ldap on the settings hooks, but we can implement it in other way.

Reply:
without this package pip fails to build `ldap` module

```bash
running build_ext
      building '_ldap' extension
      /usr/bin/ld: cannot find -lldap_r: No such file or directory
      collect2: error: ld returned 1 exit status
      error: command '/usr/bin/gcc' failed with exit code 1
```

Reply:
Also, I can't make it optional right now, because our requirements.standalone.in says that everything needed by RPM must go to setup.py, if I put djago-auth-ldap on setup.py, then it requires python-ldap, that requires openldap-devel.

Reply:
@Spredzy @fao89 @mikedep333 @simaishi 

This PR is adding the dependency on 

`openldap-devel` (Rh based systems) 

`ldap-utils ldapvi libldap-common libldap-dev python-dev libldap2-dev libsasl2-dev libssl-dev` (Debian based systems)

`libldap24` Arch based systems.

at least those were the 3 systems I have tested the install of `python-ldap`.

As Django-auth-ldap is being included as dep for galaxy_ng we need those system libraries installed on every target system.

How do we coordinate that across the installers?